### DISCUSSION TITLE
Welcome to tiny-web-crawler Discussions!

### DISCUSSION THREAD

**Discussion by indrajithi** [indrajithi/tiny-web-crawler#35] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mews** [comment]
Hi I'm mews and I just found this repo on https://goodfirstissues.com/ lol
I just enjoy contributing to open source and this is a cool project so I decided to help out :)
